src/plone/recipe/zope2instance/__init__.py

```diff
@@ -758,6 +758,15 @@ def build_package_includes(self):
             n = 1 # 001 is reserved for an optional locales-configure
             package_match = re.compile('\w+([.]\w+)*$').match
             for package in zcml:
+                if package.startswith('#'):
+                    comment_snippet = zcml[n-1:n+5]
+                    if zcml[n+5:]:
+                        comment_snippet.append('(...)')
+                    comment_snippet = ' '.join(comment_snippet)
+                    raise ValueError('Invalid comment in zcml assignment'
+                                     ' (must start in first column; something like'
+                                     ' %(comment_snippet)r)'
+                                     % locals())
```

Comment:
Nit picking: why use `locals()` here? Since the string only uses one variable, I would just say `...%r' % comment_snippet` or `'{0!r}'.format(comment_snippet)`.

Reply:
Call it lazyness; I'll do it like this most of the time. When logging, it's a comma instead of the `%` sign. Also, it's more foolproof to use a dict than a simple variable (which could continue a tuple and make the whole thing fail). Last, it should still be pretty efficient.

Reply:
Concerning jenkins, I'll need some more guidance here. I clicked over to jenkins.plone.org, and I was logged in without any futher action with my Github acount, but I can't find that "Build with parameters" form. Is this the exact name?

I had a look at https://docs.plone.org/develop/coredev/docs/intro.html#jenkins-mr-roboto as well but couldn't find a hint there.

Reply:
For Jenkins, see http://jenkinsploneorg.readthedocs.io/en/latest/run-pull-request-jobs.html. The video is helpful.

Or look at the box 'Some checks haven’t completed yet' on the github page you are currently reading. It has direct links to the relevant Plone versions that need a Jenkins run.